src/rdkafka_idempotence.c

```diff
@@ -294,11 +294,12 @@ void rd_kafka_idemp_pid_update (rd_kafka_broker_t *rkb,
  * @locks none
  */
 void rd_kafka_idemp_drain_reset (rd_kafka_t *rk) {
+        rd_kafka_wrlock(rk);
         rd_kafka_dbg(rk, EOS, "DRAIN",
-                     "Beginning partition drain for PID reset "
+                     "Beginning partition drain for %s reset "
```

Comment:
```suggestion
                     "Beginning partition drain for PID %s reset "
```

Reply:
That's bit of a weird suggestion since it revokes the change I just did? ;)
pid2str() already includes "PID":
https://github.com/edenhill/librdkafka/blob/master/src/rdkafka_proto.h#L571

Reply:
oops  :+1: